core/quantity/src/Formatter/Formatter.ts

```diff
@@ -546,4 +556,45 @@ export class Formatter {
 
     return converted.magnitude;
   }
+
+  private static formatRatio(magnitude: number, spec: FormatterSpec): string {
+    if (null === spec.format.ratioType)
+      throw new QuantityError(QuantityStatus.InvalidCompositeFormat, `The Format ${spec.format.name} must have a ratio type specified.`);
+
+    if (spec.format.hasUnits && spec.format.units && spec.format.units[0][0].phenomenon !== "Units.SLOPE") // TODO - <Naron>: this should be handled at convert?
```

Comment:
Same here, do not check for Slope because the phenomenon could have a different name based on its provider/schema